include/Core/Users/UsersModule.php

```diff
@@ -286,4 +286,27 @@ protected function render_post_type_checkboxes( $name ) {
 		<?php endforeach; ?>
 		<?php
 	}
+
+	/**
+	 * Render Tags dropdown.
+	 */
+	protected function render_user_role_dropdown() {
```

Comment:
In the issue I have specified `The helper function should be moved to Renderer class.`

You have created it to the wrong class.